_includes/author.html

```diff
@@ -6,6 +6,7 @@ <h3 class="name">
             <a itemprop="name" href="https://plus.google.com/+{{site.gplus_username}}/posts">{{site.username}}</a>
         </h3>
         <p class="desc">{{site.user_description}}</p>
-        <a itemprop="email" class="email" href="mailto:{{site.email}}">{{site.email}}</a>
+        <p><a itemprop="email" class="email" href="mailto:{{site.email}}">{{site.email}}</a></p>
+        <p><a itemprop="github" class="github" href="https://github.com/{{site.github_username}}">github.com/{{site.github_username}}</a></p>
```

Comment:
You forgot to add this github class to get theme styles, I fixed that on [8bd793](https://github.com/willianjusten/cards-jekyll-template/commit/8bd793da4d1182e82d89101a85479299f0781687)